tests/functional/u/used/used_before_assignment_typing.py

```diff
@@ -8,7 +8,62 @@
     if True:  # pylint: disable=using-constant-test
         import math
     import datetime
+    import calendar
```

Comment:
[cosmetic thing:] do you mind alphabetizing these?

Reply:
Done!